### PR TITLE
Add validation for when the logo is not loaded [Delivers #119239789]

### PR DESCRIPTION
**Pull request by chiefGui**
@chute/frontend

### CODE REVIEW

**Comment by mareksuscak** [comment]
@chiefGui Resolve conflicts first.

**Comment by chiefGui** [comment]
Done. Thanks @mareksuscak

File: src/components/live-displays/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -39,11 +45,18 @@ export default class Sidebar extends Component {
...
   renderLogo() {
+    const {isLogoLoaded} = this.state
     const {logoUrl} = this.props
 
     return (
       <section className="sidebar__logo">
-        <img className="sidebar__image" src={logoUrl} />
+        {isLogoLoaded && (
+          <img
+            ref="logo"
```
Comment (mareksuscak) [inline src/components/live-displays/Sidebar/Sidebar.jsx]: Do we need this `ref` for something specific?

Comment (chiefGui) [inline src/components/live-displays/Sidebar/Sidebar.jsx]: No, we don't. Good catch. Thanks! 👍

**Comment by mareksuscak** [comment]
👍

**Pull request merged**